Makefile

```diff
@@ -0,0 +1,123 @@
+PACKAGES_NOSIMULATION=$(shell go list ./... | grep -v '/simulation')
+PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
+export VERSION := $(shell echo $(shell git describe --always --match "v*") | sed 's/^v//')
+export TMVERSION := $(shell go list -m github.com/tendermint/tendermint | sed 's:.* ::')
+export COMMIT := $(shell git log -1 --format='%H')
+LEDGER_ENABLED ?= true
+BINDIR ?= $(GOPATH)/bin
+BUILDDIR ?= $(CURDIR)/build
+
+export GO111MODULE = on
+
+# process build tags
+
+build_tags = netgo
+ifeq ($(LEDGER_ENABLED),true)
+  ifeq ($(OS),Windows_NT)
+    GCCEXE = $(shell where gcc.exe 2> NUL)
+    ifeq ($(GCCEXE),)
+      $(error gcc.exe not installed for ledger support, please install or set LEDGER_ENABLED=false)
+    else
+      build_tags += ledger
+    endif
+  else
+    UNAME_S = $(shell uname -s)
+    ifeq ($(UNAME_S),OpenBSD)
+      $(warning OpenBSD detected, disabling ledger support (https://github.com/cosmos/cosmos-sdk/issues/1988))
+    else
+      GCC = $(shell command -v gcc 2> /dev/null)
+      ifeq ($(GCC),)
+        $(error gcc not installed for ledger support, please install or set LEDGER_ENABLED=false)
+      else
+        build_tags += ledger
+      endif
+    endif
+  endif
+endif
+
+ifeq (cleveldb,$(findstring cleveldb,$(COSMOS_BUILD_OPTIONS)))
+  build_tags += gcc
+endif
+
+whitespace :=
+whitespace += $(whitespace)
+comma := ,
+build_tags_comma_sep := $(subst $(whitespace),$(comma),$(build_tags))
+
+# process linker flags
+
+ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=cheqd-node \
+		  -X github.com/cosmos/cosmos-sdk/version.AppName=cheqd-noded \
```

Comment:
Let's move Name and AppName to constants too. Btw, what is the difference?

Reply:
Moved to constants, too.
I think the difference here, if not mistaken, is an AppName for non-separate builds for server and cli.
In case of separate builds, we have to define ServerName and ClientName.
The Name is included in both cases.

Reply:
Things were easier: https://github.com/cosmos/cosmos-sdk/blob/master/version/version.go#L27

Reply:
@ankurdotb We have two constants for `application's name` and `application binary name`.
- Binary name: Jacob suggested renaming cheqd-noded to cheqd. Do we consider changing it?
- App name: What should be there?

Reply:
No let's keep it cheqd-noded please. Since the name is cheqd and the default is to add a "d" at the end, I don't want to be "cheqdd".